### PR TITLE
Feature/x4 publibs 7 add new feature din

### PR DESCRIPTION
**Pull request by anonaka**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -90,3 +90,38 @@ You can send sms or make a phone call with just a few line of python code.
 
 
 3. You can check the call status by get_simple_playback_status() method. You will provide call-id of the call you want to check.
+
+*Check SMS sent status*::
+
+ xoxzo_res = self.xc.get_sent_sms_list(sent_date=">=2016-13-01")
```
Comment (k4ml) [inline README.rst]: Do you need `self` here ?

File: README.rst [hunk 2]
```diff
@@ -90,3 +90,38 @@ You can send sms or make a phone call with just a few line of python code.
...
+
+*Explanation*
+
+You can check sent SMS status spcifying a certain date. You can use comparison operators such as "<=,<,=,>,>="
```
Comment (k4ml) [inline README.rst]: Typo - spcifying.

Comment (anonaka) [inline README.rst]: fixed.

File: README.rst (hunk 1 above)
Comment (anonaka) [inline README.rst]: fixed.

**Comment by anonaka** [comment]
This is wrong PR  to master branch. Should be PR to the develop branch. Closing.

**Pull request closed without merging**